src/serialization.cpp

```diff
@@ -85,7 +86,13 @@ Status serializePredictResponse(
             return status;
         }
         auto& tensorProto = (*response->mutable_outputs())[networkOutput->getMappedName()];
-        auto status = serializeBlobToTensorProto(tensorProto, networkOutput, blob);
+        Status status;
+        std::string binary_suffix = "_bytes";
```

Comment:
This should not be variable. Make it const static and move to some utility header. Check implementation of ModelInstance::waitForLoaded to check how constants are handled there.